packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 3.2.0
```

Comment:
Could you rebase this PR on master and adapt the changelog according to the new format we are using?

Reply:
Yes, but where do I see the example format?